modules/va_notify/app/workers/va_notify/clear_stale_in_progress_reminders_sent.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module VANotify
+  class ClearStaleInProgressRemindersSent
+    include Sidekiq::Worker
+
+    def perform
+      InProgressRemindersSent.destroy_by('created_at < ?', 60.days.ago)
```

Comment:
I know we didn't explicitly mention this in the ticket but I think it would be good to have a feature flag on this job. That way if we need to turn it off for some reason it won't require another deploy.